_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -2482,7 +2482,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
     }
 
     bool laEnabled = true;
-    if (platform > MFX_HW_ICL_LP)
+    if (platform > MFX_HW_ICL_LP && platform != MFX_HW_TGL_LP)
```

Comment:
Please add a comment why we need such condition

Reply:
Okay, it's VME dependency. Please name this accordingly. Please also check HEVC part. The shall the same condition there.